### PR TITLE
Update specifications to build project

### PR DESCRIPTION
**Pull request by cacarrara**
Three commits that update specifications of node's version, ruby dependencies (sass and compass) and some new instructions to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +17,7 @@ Frontend app para nova versão do atados.com.br programado com AngularJS.
 	sudo npm install -g grunt-cli
 	npm install
 	bower install
+    bundle
```
Comment (yokomizor) [inline README.md]: You should replace spaces for tab here.

**Comment by yokomizor** [comment]
LGTM (:

**Comment by cacarrara** [comment]
Done. Thanks! (:

**Comment by mpomarole** [comment]
super like!

**Pull request merged**